src/main/resources/com/google/api/codegen/java/grpc_package.snip

```diff
@@ -0,0 +1,138 @@
+@snippet generate(metadata)
```

Comment:
It's fine to reconcile that, but you're missing other things like the grpc dependencies and generation of javadocs.